src/main/java/com/uid2/client/PublisherUid2Client.java

```diff
@@ -62,16 +61,28 @@ public TokenGenerateResponse generateTokenResponse(TokenGenerateInput tokenGener
 
 
         try (Response response = client.newCall(request).execute()) {
+            final String responseString = getResponse(response);
+            
             if (!response.isSuccessful()) {
-                throw new Uid2Exception("Unexpected code " + response);
+                throw new Uid2Exception("Unexpected code " + responseString);
```

Comment:
could the above 2 lines could go in the method as well?

Reply:
done